sdk/loadtestservice/Azure.ResourceManager.LoadTesting/CHANGELOG.md

```diff
@@ -1,15 +1,20 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2023-01-06)
 
 ### Features Added
 
+- Update to the latest api-version.
```

Comment:
Did we update the API version? It seems that the beta.1 has already used the API version `2022-12-01`

Reply:
Yes, service team has already deployed the stable version for public preview